### DISCUSSION TITLE
add single/multiple weekdays to charging schedule

### DISCUSSION THREAD

**Discussion by owkowo69** [pkuehnel/TeslaSolarCharger#1607] [Ideas]
Hi,

i'm new to TSC and would like to use PV+Spot Mode.
It would be helpful if i can set different MinSoc for specific Weekdays, as i need e.g. 100% SOC on fridays.

thanks for that great project.
Wolfgang

**Comment by pkuehnel** [comment]
Added to the Backlog (https://github.com/pkuehnel/TeslaSolarCharger/issues/1608#issue-2645986007) but currently many other issues that need to be resolved before that. Note: Everything you can do in the UI, you can automate via REST API if you want. Just add `/swagger` to your TSC URL and you can see all REST API endpoints.

**Reply by owkowo69** [reply]
Thanks for that hint, i'll give it a try.
